tests/fixtures/rules/no-trailing-zeros/invalid/input.yml

```diff
@@ -0,0 +1,12 @@
+# {"options": []}
+-   {
+      a: 1.23,
+      b: 1.20,
+      c: .23,
+      d: .20
+    }
+-
+    - "a": 1.23
+    - "b": 1.20
+    - "c": .23
+    - "d": .20
```

Comment:
Could you please add test cases to test `1.0` and `.0`?

Reply:
I just noticed that we also need to add tests involving signs. e.g. `+1.0`, `-1.0`